vernac/declareDef.ml

```diff
@@ -38,7 +40,8 @@ let declare_definition ident (local, p, k) ce pl imps hook =
   let gr = match local with
   | Discharge when Lib.sections_are_opened () ->
       let _ = declare_variable ident (Lib.cwd(), SectionLocalDef ce, IsDefinition k) in
-      let () = if Proof_global.there_are_pending_proofs () then warn_definition_not_visible ident in
+      (* XXX *)
+      (* let () = if Proof_global.there_are_pending_proofs () then warn_definition_not_visible ident in *)
```

Comment:
Is there a particular problem to keep this code?

Reply:
Nope, even tho the check could be surely moved; fixed!

Reply:
Fixed.